src/injection/react-native-web.js

```diff
@@ -10,7 +10,7 @@ const {
   Dimensions,
   Easing,
 } = require('react-native-web');
-const StyleRegistry = require('react-native-web/dist/apis/StyleSheet/registry');
+const StyleRegistry = require('react-native-web/dist/apis/StyleSheet/registry').default;
```

Comment:
1) this should probably do the require, and then do `x.default || x` - rather than restricting the range.
2) this is a bug on react-native-web - babel transpilation details shouldn't be leaking out to the package. They should add the `add-module-exports` transform.

Reply:
I am open for changes, but I would like to know why @lelandrichardson choosed to use require over import statements. If I am adding an import where the entire codebase use require, it will be a bit weird.